sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -27,6 +27,8 @@
 if TYPE_CHECKING:
     from datetime import datetime
 
+PUBLIC_AUDIENCE = "https://storage.azure.com/.default"
```

Comment:
Do you know if it's possible to make a class constant? I tried but doesn't seem like it. We also already have this defined as a constant, but that constant file lives in `_shared` which is a directory down from this layer so had to re-define it here.

If we decide to leave each package's Audience in it's respective `_models.py`, we have no choice but to re-define it.

Reply:
Can't you just do `from ._shared.constants import STORAGE_OAUTH_SCOPE`?